sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultServiceSpec.java

```diff
@@ -74,6 +80,9 @@ public DefaultServiceSpec(
                 ? DEFAULT_ZK_CONNECTION : zookeeperConnection;
         this.pods = pods;
         this.replacementFailurePolicy = replacementFailurePolicy;
+        // If we're using the YAML then gpuFlag should never be null, however it's possible for someone to
```

Comment:
This behavior is common throughout the SDK and need not be specifically called out here.